index.html

```diff
@@ -144,6 +144,6 @@ <h1>The end</h1>
         </section>
       </div>
     </div>
-    <script src="dist/app.js"></script>
+    <script src="assets/js/app.js"></script>
```

Comment:
Changed this to `assets/...` since the index file will be in the dist folder and all paths are relative from there.